### ISSUE TITLE
Table - accessible examples

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1460]
There are several cases where our examples don't represent best practice with regard to accessibility.

- **aria-labelledby attributes on the checkboxes and expand buttons**
  - **html elements affected:**
    - `<input aria-labelledby="simple-node0" name="checkrow0" type="checkbox">`
    -  `<button aria-labelledby="simple-node1 expandable-toggle1" id="expandable-toggle1" aria-label="Details" aria-expanded="true" class="pf-c-button pf-m-plain pf-m-expanded" type="button">`
  - **issue:** `aria-labelledby` is defined, but the value doesn't exist as an `id` on the first content cell
  - **update:** In previous versions of these examples (at least as far back as https://github.com/patternfly/patternfly-react/pull/1324, maybe sooner), this was handled in the following way:
    ```
    <th data-label="Repositories" data-key="1">
      <div id="simple-node0">one</div>
    </th>
    ```
- **Examples showing the action button should include row headers**
  - One of the accessibility benefits for having a row header is that the row header provides context for other cell contents in that row. This is especially important for examples where the Actions button is included, so that the screen reader user knows which item is being affected by the actions in the kebab. There is a react example that illustrates how to define a row header—“First cell as Header”. This same method should be used in the react examples that show action buttons.
